### ISSUE TITLE
Snowflake string check

### ISSUE THREAD

**Issue by DrTTrousers** [OHDSI/CohortGeneratorModule#15]
``` r
start target CohortGeneratorModule_1
Validating inputs
Executing
Connecting using Snowflake driver
Creating cohort tables
An error report has been created at  ~/strategusOutput/CohortGeneratorModule_1/errorReport.R
Error in execute(jobContext) : 1 assertions failed:
 * Variable 'targetDialect': Must be element of set {'sql
 * server','oracle','postgresql','pdw','impala','netezza','bigquery','spark','sqlite','redshift','hive','sqlite
 * extended'}, but is 'snowflake'.
Calls: execute -> <Anonymous> -> <Anonymous> -> <Anonymous>
Execution halted
✖ error target CohortGeneratorModule_1
• end pipeline [14.567 seconds]
Error:
! Error running targets::tar_make()
  Error messages: targets::tar_meta(fields = error, complete_only = TRUE)
  Debugging guide: https://books.ropensci.org/targets/debugging.html
  How to ask for help: https://books.ropensci.org/targets/help.html
  Last error: Module did not complete. To debug:
```

Compiling the guinea pig real application script for Strategus (Ref https://github.com/OHDSI/Strategus/issues/26 ) and have discovered that there is a hard text string check inside this module that doesn't like snowflake.

CohortGenerator itself doesn't seem to mind snowflake, I've successfully run several packages using it on snowflake already. I think this hard check needs removing.

Happy to provide testing input on the linux env and snowflake.

**Comment by anthonysena** [comment]
@DrTTrousers thanks for reporting this - this likely require an update of DatabaseConnector & SqlRender for the module to support Snowflake.

**Comment by anthonysena** [comment]
This should be fixed as the latest release (v0.2.0) uses the latest DatabaseConnector and SqlRender packages.
